sdk/identity/azure-identity/README.md

```diff
@@ -11,32 +11,8 @@ The Azure Identity library provides Azure Active Directory token authentication
 
   [Source code][source] | [API reference documentation][javadoc] | [Azure Active Directory documentation][aad_doc]
 
-## Table of contents
```

Comment:
Why is this removed? Not required anymore, doesn't conform to README standards?

Reply:
This was removed based on arch board feedback